README.rst

```diff
@@ -277,6 +277,9 @@ Ubuntu and derivatives
 - KDE neon (based on Ubuntu 16.04)
 - Linux Mint 17/18
 - Ubuntu 14.04/16.04 and subsequent non-TLS releases (see below)
+- Python3 (see ``-x`` option):
+
+  - Ubuntu 16.04
```

Comment:
This looks little bit confusing... I think we could add separate section to the README covering a status of Py3 support.

Reply:
That's a better idea. I will fix that. Thanks!